dev/io.openliberty.jakarta.validation.v31_fat/test-applications/val31/src/val31/web/Validation31TestServlet.java

```diff
@@ -62,19 +64,30 @@ public void recordValidatePropertyAndValueTest() throws Exception {
     }
 
     /**
-     * Test that the result of a method on a record can be validated.
+     * This tests if able to validate parameters and return value with records.
      */
-
     @Test
-    public void recordMethodValidationTest() throws Exception {
-        // TODO Add a method that includes validation to a record class and confirm that it can be validated
+    public void validateRecordParameters() throws Exception {
```

Comment:
Test methods should end with Test, so `validateRecordParametersTest` would be better

Reply:
Updated